os/arch/arm/src/imxrt/imxrt_mtd_flash.c

```diff
@@ -138,7 +138,7 @@ struct imxrt_dev_s {
 static inline int imxrt_readid(struct imxrt_dev_s *priv);
 static void imxrt_waitwritecomplete(struct imxrt_dev_s *priv);
 static inline void imxrt_sectorerase(struct imxrt_dev_s *priv, off_t offset);
-static inline int imxrt_bulkerase(struct imxrt_dev_s *priv, unsigned long arg);
+static inline int imxrt_bulkerase(struct imxrt_dev_s *priv, size_t startsector, size_t nsectors);
```

Comment:
Please check second parameter name,
Where implemented, it is defined as `size_t startblock`.

Reply:
Thx, modified.